src/ui/components/CreateIdentifier/components/IdentifierStage1BodyResume.tsx

```diff
@@ -140,16 +141,21 @@ const IdentifierStage1BodyResume = ({
           </IonList>
         )}
       </ScrollablePageLayout>
-      {groupMetadata?.groupInitiator && (
-        <PageFooter
-          pageId="initiate-multi-sig"
-          primaryButtonText={`${i18n.t(
-            "createidentifier.share.initiatebutton"
-          )}`}
-          primaryButtonAction={handleInitiateMultiSig}
-          primaryButtonDisabled={!scannedConections?.length}
-        />
-      )}
+      <PageFooter
+        pageId="initiate-multi-sig"
+        customClass={
+          groupMetadata?.groupInitiator ? "stage-1-footer" : undefined
+        }
+        primaryButtonText={
+          groupMetadata?.groupInitiator
+            ? `${i18n.t("createidentifier.share.initiatebutton")}`
+            : undefined
+        }
+        primaryButtonAction={handleInitiateMultiSig}
+        primaryButtonDisabled={!scannedConections?.length}
+        deleteButtonAction={handleDelete}
```

Comment:
The delete button should be added to the page, no need to be position fixed at the bottom of the screen.


See demo of the design for clarity.


https://github.com/user-attachments/assets/5bbf4e50-af73-48b2-8538-df4fa8fcccb1